pkgs/top-level/all-packages.nix

```diff
@@ -1893,6 +1893,8 @@ with pkgs;
   };
   doomseeker = callPackage ../applications/misc/doomseeker { };
 
+  doom-bcc-git = callPackage ../games/zdoom/bcc-git.nix { };
```

Comment:
Please remove `-git` suffix here. We use `Unstable` if we have both version of the software.